terraform/modules/bosh_vpc/vpc.tf

```diff
@@ -13,4 +13,59 @@ resource "aws_vpc" "main_vpc" {
     Name = "${var.stack_description}"
   }
 
-}
\ No newline at end of file
+}
+
+# Create CloudWatch log group
+resource "aws_cloudwatch_log_group" "main_vpc_flow_log_cloudwatch_log_group" {
+  name = "${var.stack_description}-flow-log-group"
+}
+
+# Create IAM role for sending flow logs
+resource "aws_iam_role" "flow_log_role" {
+    name = "${var.stack_description}-flow-log-role"
+    assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "vpc-flow-logs.amazonaws.com"
```

Comment:
@dlapiduz this isn't some odd thing that may be different for govcloud is it? Also, if it is, when you change it, make sure you leave a TODO in a comment to come back and make this configurable.